alkira/resource_alkira_connector_gcp_vpc.go

```diff
@@ -40,6 +40,35 @@ func resourceAlkiraConnectorGcpVpc() *schema.Resource {
 				Optional:    true,
 				Default:     true,
 			},
+			"gcp_routing": {
+				Description: "GCP Routing describes the routes that are to be imported to the VPC " +
+					"from the CXP. This essentially controls how traffic is routed between the " +
+					"CXP and the VPC. gcpRouting provides a customized routing specification. " +
+					"When gcpRouting is not provided i.e when gcpRouting is null/empty then all " +
+					"traffic exiting the VPC will be sent to the CXP (i.e a default route to " +
+					"CXP will be added to all route tables on that VPC)",
+				Type: schema.TypeSet,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"prefix_list_ids": {
+							Description: "Ids of prefix lists defined on the network.",
+							Type:        schema.TypeList,
+							Required:    true,
+							Elem:        &schema.Schema{Type: schema.TypeInt},
+						},
+						"custom_prefix": {
+							Description: "The routeImportMode is an instruction which specifies " +
+								"the source of the routes that need to be imported. Only " +
+								"`ADVERTISE_DEFAULT_ROUTE` and `ADVERTISE_CUSTOM_PREFIX` are valid inputs.",
+							Type:             schema.TypeString,
+							Required:         true,
+							ValidateDiagFunc: validateCustomPrefix,
```

Comment:
This could be done directly like this:

```
ValidateFunc: validation.StringInSlice([]string{"ADVERTISE_DEFAULT_ROUTE", "ADVERTISE_CUSTOM_PREFIX"}, false),
```